apps/web/src/app/page.tsx

```
@@ -0,0 +1,27 @@
+import { Loading } from "@/components/loading";
+import { getAllLocations } from "@/db/locations";
+import { Suspense } from "react";
+import Map from "./map";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)